.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
+}
```

Comment:
Please use the linter[ config file provided by microverse](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json).....then re-run for linting errors and you get a thumbs. Thats all what's left to do.
[[see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png)]